pika/adapters/tornado_connection.py

```diff
@@ -72,7 +73,10 @@ def _adapter_disconnect(self):
         """Disconnect from the RabbitMQ broker"""
         if self.socket:
             self.ioloop.remove_handler(self.socket.fileno())
-        super(TornadoConnection, self)._adapter_disconnect()
+        try:
+            super(TornadoConnection, self)._adapter_disconnect()
+        except exceptions.AMQPConnectionError as e:
```

Comment:
Is the exception being intentionally suppressed here? Please add a comment explaining why the exception is being suppressed and how user code would become informed of the failure.